plugins/ExamplePlugin/tests/test_example_plugin.xml

```diff
@@ -69,24 +94,4 @@
     </PointSet>
   </DataObjects>
 
-  <Steps>
-    <MultiRun name="MCrun">
-      <Input          class="DataObjects"   type="PointSet">sumOfExponentialInput</Input>
-      <Model          class="Models"        type="ExternalModel">sum_exp</Model>
-      <Sampler        class="Samplers"      type="MonteCarlo">test_MC</Sampler>
-      <Output         class="DataObjects"   type="PointSet">sumOfExponentialOutput</Output>
-    </MultiRun>
-    <IOStep name="printTOfile">
-     <Input          class="DataObjects"   type="PointSet">sumOfExponentialOutput</Input>
-     <Output         class="OutStreams"    type="Print">dumpSumOfExpOutput</Output>
-    </IOStep>
-  </Steps>
-
-  <OutStreams>
-    <Print name="dumpSumOfExpOutput">
-      <type>csv</type>
-      <source>sumOfExponentialOutput</source>
-      <what>input,output</what>
-    </Print>
-  </OutStreams>
```

Comment:
FYI, rearranging this so that OutStreams loaded before Models makes sure the non-ExternalModel loading works correctly; otherwise ExternalModel forces the plugin loading without checking the OutStreams/Generic mechanics.

Reply:
Do you have a solution for this without rearranging?

Reply:
I am open to suggestions; I do not have another solution right now. This seems like a decent solution, though.